jaqpotpy/api/jaqpot_api_client.py

```diff
@@ -193,7 +219,33 @@ def predict_with_csv_sync(self, model_id, csv_path):
             status_code=response.status_code.value,
         )
 
+    def get_dataset(self, response):
+        """
+        Retrieves a dataset using the location provided in the response headers.
+        Args:
+            response (requests.Response): The HTTP response object containing the dataset location in the headers.
+        Returns:
+            dict: The dataset retrieved by its ID.
+        """
+
+        dataset_location = response.headers["Location"]
```

Comment:
move this code to the predict_async function

Reply:
You mean that predict_async should return the dataset location instead of response?